api/handlers.go

```diff
@@ -71,6 +83,18 @@ func (api *API) unpinPOST(w http.ResponseWriter, req *http.Request, _ httprouter
 		api.WriteError(w, database.ErrInvalidSkylink, http.StatusBadRequest)
 		return
 	}
+	if sl.IsSkylinkV2() {
```

Comment:
follow up; I would add a `parseSkylink` helper to ensure all endpoints and future endpoint parse the skylink the same way.

Reply:
https://github.com/SkynetLabs/pinner/issues/12